File: SS1/Ex5/CPP/Ex5.cpp
// #include <stdio.h>
// #include <stdlib.h>
// #include <time.h>
 
// int main(){
//     int r;
//     srand((int)time(0));
//     for(int i = 0; i < 5; ++i){
//         r = rand();
//         printf("%d\n", r);
//     }    
// }

#include<iostream>
#include<conio.h>
#include <ctime>
#include <cstdlib>
#include<windows.h>
#include <math.h>
using namespace std;
void maxRand(int a[],int n){
	for(;;)
	{
		int r2 = static_cast <int> (rand()) / (static_cast <int> (RAND_MAX/1000));
		cout<<r2<<endl;
		for(int i = 0 ;i<n ;i++){
			if(a[0] == 0)
			{
				a[0]=a[1];
				n--;
			}
			a[n]=r2;
		}
		n=n+1;
		int tg;
		for(int i=0;i<n;i++){
			for(int j=i+1;j<n;j++)
			{
				if(a[i] < a[j]){
	                tg = a[i];
	                a[i] = a[j];
	                a[j] = tg;        
	            }
			}	
		}
		cout<<"3 So Lon Nhat: "<<a[0]<<","<<a[1]<<","<<a[2];	
		cout<<endl;
		cout<<endl;
		Sleep(1000);
	}
}

int main()
{
	int a[10000];
	maxRand(a,1);
}

File: SS1/Ex5/Java/Ex5.java
import java.util.Random;

public class Ex5 {
    public static void maxRand(int []a,int n){
        while(true)
        {
            Random r2 = new Random();
            int value = r2.nextInt();
            System.out.println(value);
           for (int i = 0; i < n; i++)
            {
                if (a[0] == 0)
		{
                    a[0] = a[1];
		    n--;
		}
		a[n] = value;
            }
            n = n + 1;
            int tg;
            for (int i = 0; i < n; i++)
	    {
		for (int j = i + 1; j < n; j++)
		{
		    if (a[i] < a[j])
                    {
			tg = a[i];
			a[i] = a[j];
			a[j] = tg;
		    }
		}
	    }
            System.out.println("3 So tri lon nhat: " + a[0] + "," + a[1] + "," + a[2]);
            try{
                Thread.sleep(5000); 
            }
             catch (InterruptedException e) {
                System.out.println(e);
            }
        }
    }

    public static void main(String[] args)
    {
        int[] a = new int[100];
        maxRand(a,1);
    }
}